### ISSUE TITLE
test:debug not working?

### ISSUE THREAD

**Issue by shadowii** [CodingZeal/react-boilerplate#58]
I'm getting `test.js:9033 Uncaught TypeError: Cannot delete property '__REACT_DEVTOOLS_GLOBAL_HOOK__' of #<Window>` errors. 

Also, the `test:debug` script no longer seems to specify a webpack config file.

**Comment by randycoulman** [comment]
We've been seeing that error lately as well.  It is somehow tied to the React devtools, and we haven't gotten to the bottom of it yet.  The only workaround we know of at this point is to temporarily disable the React dev tools when running tests in the browser.

I'll leave this issue open as a reminder to try to get to the bottom of the issue.

**Comment by randycoulman** [comment]
See #37 for more in the second item you mention.  We now specify `NODE_ENV=test-debug` when running `npm run test:debug`, and the main webpack configuration file (`webpack.config.js`) choses the appropriate webpack configuration file based on the environment.  This was needed to get `eslint-plugin-import` to work correctly.

**Comment by randycoulman** [comment]
I did some additional digging into this issue today.  It turns out that the root cause is [this issue](https://github.com/bruderstein/react-render-hook/issues/4) on react-render-hook, which is ultimately used by expect-react-shallow.  It is possible that the underlying issue can now be addressed given work that was done in React 15.1. Until react-render-hook has been updated accordingly, we're stuck with the following workarounds:

1. Disable react-devtools when debugging tests in the browser.
2. Stop using expect-react-shallow.
